include/range/v3/view/istream.hpp

```diff
@@ -112,7 +112,7 @@ namespace ranges
 
     namespace cpp20
     {
-        template<typename Val>
+        template<typename Val, class CharT = char, class Traits = std::char_traits<CharT>>
```

Comment:
I'd rather we leave this alone until we make a conforming `basic_istream_view`. Probably a separate PR.